rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -546,7 +544,8 @@ def migrate_to_v3(connection: 'DBConnection') -> None:
         # database and are not EVM (matic tokens, Otimism tokens, etc) + their variants in
         # other chains. And populates them properly via sql statements
         with open(dir_path / 'data' / 'globaldb_v2_v3_assets.sql', 'r') as f:
-            sql_sentences = f.read()
-            cursor.executescript(sql_sentences)
-        connection.commit()
+            raw_sql_sentences = f.read()
+            per_table_sentences = raw_sql_sentences.split('\n\n')
+            for sql_sentences in per_table_sentences:
```

Comment:
The executescript here was used since that sql file had 3 insert for the three tables that contain information about the evm assets. What would be nice is to revert the changes if something goes wrong for some reason since otherwise the state becomes inconsistent. (say that you set information in 2 of the 3 tables for some reason). In that case we should rollback i guess